pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -64,6 +64,11 @@
       <TargetFramework>.NETCoreApp1.1</TargetFramework>
     </Dependency>
 
+    <!-- references the host packages -->
+    <Dependency Include="Microsoft.NETCore.DotNetHostPolicy">
```

Comment:
Is there a reason this reference is a Dependency as opposed to P2P reference like https://github.com/dotnet/core-setup/blob/release/1.1.0/pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj#L11, which was removed in https://github.com/dotnet/core-setup/commit/81e058c627c963ac996177b36a2ab8ecd66a8095#diff-e2c52444a9da4a5eae65d36e5d3e6aba?